templates/insurance_companies/general_list.html.twig

```diff
@@ -0,0 +1,62 @@
+<html>
+    <head>
+        {{ setupHeader(['common','datatables','datatables-colreorder','datatables-dt','datatables-bs']) }}
+        <title>{{ "Insurance Companies"|xlt }} {{ applicationTitle|text }}</title>
+    </head>
+<body>
+<a href="{{webroot|attr_url}}/controller.php?practice_settings&insurance_company&action=edit" onclick="top.restoreSession()" class="btn btn-secondary btn-add">{{'Add a Company'|xlt}}</a>
```

Comment:
only need to do attr on the webroot. the url may cause weirdness (not really sure, but not something have done in past, since generally just use url to encode the parameters to allow special things like & and ? without breaking the parameters)